resources/Main.qml

```diff
@@ -36,120 +36,124 @@ ApplicationWindow {
     }
 
     Item {
-        anchors.centerIn: base.contentItem
-        width: _portrait ? base.height : base.width
-        height: _portrait ? base.width : base.height
-        rotation: _portrait ? 90 : 0
-
-        Background {
-            anchors.fill: parent
-            rotateCompass: menuArea.expanded
-        }
+        anchors.fill: parent
 
-        AppPage {
-            id: pagesArea
-            width:  parent.width*1.4
-            height: parent.height
-            widthOffset: width - parent.width + 80
-            x: menuArea.expanded ? parent.width : 80
-            title: menuArea.currentItemText
+        Item {
+            anchors.centerIn: parent
+            width: _portrait ? parent.height : parent.width
+            height: _portrait ? parent.width : parent.height
+            rotation: _portrait ? 90 : 0
 
-            SwipeView {
+            Background {
```

Comment:
I just added another Item to fill the parent, because wasm was not starting at the right size (only after a click it noticed the window size), with this it just works